webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/ncf/TechniqueReader.scala

```diff
@@ -45,6 +47,9 @@ class TechniqueReader(
   val methodsFile = ncfRootDir / "generic_methods.json"
 
 
+  val currentTimeMillis = ZIO.accessM[_root_.zio.clock.Clock](_.get.currentTime(TimeUnit.MILLISECONDS)).provide(ZioRuntime.environment)
+  val currentTimeNanos  = ZIO.accessM[_root_.zio.clock.Clock](_.get.nanoTime).provide(ZioRuntime.environment)
```

Comment:
debug things ? 
For info, currenttimemillis is available in our `ZioRuntime`, and for debug in effectful parts, you can just use jvm methods, zio one does not provide anything else

Reply:
yes it was to time what was taking time. I can remove them